runtime/src/mailbox_api.rs

```diff
@@ -264,3 +268,11 @@ pub struct FwInfoResp {
     // TODO: Decide what other information to report for general firmware
     // status.
 }
+
+/// SVN response
+#[repr(C)]
+#[derive(Debug, AsBytes, FromBytes, PartialEq, Eq)]
+pub struct GetSvnResp {
+    pub runtime_svn: u32,
+    pub fmc_manifest_svn: u32,
+}
```

Comment:
Instead of making a new command, I think we should add these to the FW_INFO command. Thanks!

Reply:
Done!